src/components/share-panel/share-panel.tsx

```diff
@@ -63,16 +72,19 @@ export const SharePanel: React.FC = () => {
             <IconEmail />
           </EmailShareButton>
         </SocialLinkItem>
-        <SocialLinkItem>
-          <FacebookShareButton url={shareUrl} quote={title}>
-            <IconFacebook />
-          </FacebookShareButton>
-        </SocialLinkItem>
         <SocialLinkItem>
           <TwitterShareButton url={shareUrl} title={title} hashtags={hashtags}>
             <IconTwitter color="#668cff" />
           </TwitterShareButton>
         </SocialLinkItem>
+        <SocialLinkItem>
+          <LinkedinShareButton title={title} url={shareUrl}>
+            <IconLinkedIn color="#668cff" />
+          </LinkedinShareButton>
+        </SocialLinkItem>
+        <SocialLinkItem>
+          <IconXing color="#668cff" />
```

Comment:
Nothing happens if I click on it 🧐

Reply:
there is no implementation yet.
Perhaps, have you any suggestion for XING-sharing?

Reply:
```
https://www.xing.com/social/share/spi?url={URL}
```

Example: https://www.xing.com/social/share/spi?url=https%3A%2F%2Fsatellytes.com